scripts/prepare_conv_pp.py

```diff
@@ -0,0 +1,109 @@
+# coding: utf-8
+r"""
+Inputs
+-------
+in_path1 : str
+    path of input file with raw opsd data as .csv
+in_path2 : str
+    path of input file with geodata of regions in Germany as .gpgk
+in_path3 : str
+    path of input file with names of regions in Berlin and Brandenburg as .yaml
+out_path : str
+    path of output file with prepared data as .csv
+
+Outputs
+---------
+pandas.DataFrame
+    with grouped and aggregated data of conventional power plants in Berlin and Brandenburg.
+    Data is grouped by region, energy source, technology and chp capability and contains
+    net capacity and efficiency.
+
+Description
+-------------
+The script filters the OPSD conventional power plant package for power plants
+in Berlin and Brandenburg. The retrieved data is stored in a new dataframe, aggregated and
+saved as a csv file.
+Only operating power plants are considered.
+"""
+import sys
+import pandas as pd
+import yaml
+from os.path import dirname, abspath
+
+d = dirname(dirname(abspath(__file__)))
+
+sys.path.append(d)
+
+import oemof_b3.tools.geo as geo
```

Comment:
Did you encounter a problem importing here? Could be because the module oemof_b3 did not include __init__.py on all levels. Tried to fix with

Reply:
Yes, if I run the script, oemof_b3 cannot be found:
```
Traceback (most recent call last):
  File "C:\Users\meinm\Documents\Git\oemof-B3\scripts\prepare_conv_pp.py", line 33, in <module>
    import oemof_b3.tools.geo as geo
ModuleNotFoundError: No module named 'oemof_b3'
```
This Error still occurs.

Reply:
Did you install oemof_b3 in your environment?

Reply:
Ah no, how do I do it?
pip install oemof_b3 does not work.

Reply:
try this:
pip install -e <path to oemof-B3>

Reply:
This did not work. During the installation process something went wrong.
I tried ```python setup.py install``` instead, but running the script it still produces the same Error.

Reply:
Can you post the error you get during installation?

Reply:
``` 
ERROR: Command errored out with exit status 1: 'C:\Users\meinm\anaconda3\envs\rli_umas\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\meinm\\AppData\\Local\\Temp\\pip-install-r70u9c8w\\pandas_e525b264add04ae9ad6301a9eeb6d0e6\\setup.py'"'"'; __file__='"'"'C:\\Users\\meinm\\AppData\\Local\\Temp\\pip-install-r70u9c8w\\pandas_e525b264add04ae9ad6301a9eeb6d0e6\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\meinm\AppData\Local\Temp\pip-record-ubcvsyb3\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\meinm\anaconda3\envs\rli_umas\Include\pandas' Check the logs for full command output.
```

Reply:
After installing Cython in the virtual environment I am using for oemof, everything works now. Apparently the virtual env does not see the "global" Cython as indicated in https://github.com/pandas-dev/pandas/issues/3785 
Then, this Error appeared:
`FileNotFoundError: [WinError 2] Das System kann die angegebene Datei nicht finden: 'C:\\Users\\meinm\\anaconda3\\envs\\rli_umas\\lib\\site-packages\\oemof_b3-0.0.0-py3.9.egg'`
But after running the installation a couple of times, the Error disappeared and I now have omoef-0.3.2-py3.9.egg installed on my computer.

Reply:
In my local oemof-B3 git folder are now two new folders: build/ and dist/ 
dist/ includes the necessary oemof_B3-0.0.0-py3.9.egg